doc/source/security/encrypting-redo-log.rst

```diff
@@ -0,0 +1,37 @@
+.. _encrypting-redo-log:
+
+================================================================================
+Encrypting the Redo Log
+================================================================================
+
+The Redo log can be encrypted with the :variable: `innodb_redo_log_encrypt` variable. The default value for the variable is ``OFF``. The Redo log uses the tablespace encryption key.
```

Comment:
What's the "the tablespace encryption key"?

* With the MASTER_KEY option, redo log has its own tablespace key for encryption, like normal tablespaces
* With KEYRING_KEY, it has a versioned key, which can be changed